packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -77,6 +77,16 @@ const MANAGER = new CurlyComponentManager();
 import { ComponentDefinition, ValueReference } from 'glimmer-runtime';
 import Component from '../ember-views/component';
 
+function tagName(vm) {
+  let { tagName } = vm.dynamicScope().view;
+
+  if (tagName === '') {
+    throw new Error('Not implemented: fragments (`tagName: ""`)');
+  }
+
+  return new ValueReference(tagName || 'div');
```

Comment:
Confirm. We are just throwing (on line 84) for now so that whoever running into this will not be confused